docs/KNOWNISSUES.md

```diff
@@ -201,6 +206,13 @@ The following table lists commands that are known not to work in PowerShell on L
 </tr>
 </table>
 
+Installing Software using PackageManagement and PowerShellGet Modules
```

Comment:
@TravisEz13 Yes, this issue got fixed in the latest builds. This is just a known issue for https://github.com/PowerShell/PowerShell/releases/tag/v6.0.0-alpha.9 release.